dist/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <link rel="stylesheet" href="./modal.css" type="text/css" />
-    <link rel="stylesheet" href="./arrow.css" type="text/css" />
-    <link rel="stylesheet" href="./styles.css" type="text/css" />
+    <link rel="stylesheet" href="../client/modal.css" type="text/css" />
+    <link rel="stylesheet" href="../client/arrow.css" type="text/css" />
+    <link rel="stylesheet" href="../client/styles.css" type="text/css" />
```

Comment:
With a proper express server setup to serve the dist folder, '../client' is not valid. Your css files should be in the same folder as your index.html or inside a subdirectory inside the same directory as your index.html. But it cannot be located outside of the index.html's directory.